### PR TITLE
feat(state_network): implement enr and node_id endpoints for state network

### PR DESCRIPTION
**Pull request by morph-dev**
### What was wrong?

State network doesn't have any endpoint implemented.

### How was it fixed?

I added implementation for the enr and node_id related queries.

Next steps are:

- Implement `StateStorage` (at least minimal support for now)
- Implement the rest of the endpoints

### To-Do

- [ ] Clean up commit history and use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).

### CODE REVIEW

File: ethportal-api/src/types/jsonrpc/endpoints.rs [hunk 1]
```diff
@@ -14,42 +14,42 @@ pub enum Discv5Endpoint {
 /// State network JSON-RPC endpoints. Start with "portal_state" prefix
 #[derive(Debug, PartialEq, Eq, Clone)]
 pub enum StateEndpoint {
```
Comment (morph-dev) [inline ethportal-api/src/types/jsonrpc/endpoints.rs:16]: I reordered these so they are grouped by similar logic: enr/node related, content related

**Review by njgheorghita — COMMENTED** [review]
@morph-dev Any thoughts on adding some tests to `ethportal-peertest`? Seems like simply copying some history endpoints in [basic.rs](https://github.com/ethereum/trin/blob/master/ethportal-peertest/src/scenarios/basic.rs) is a good place to start. There's certainly some opportunity for refactoring, since I don't believe `ethportal-peertest` was updated throughout the beacon network implementation. However, I don't feel too strongly about refactoring `ethportal-peertest` but these seem like endpoints that should be tested. I'm also happy if you want to do it in a following pr

**Review by ogenev — APPROVED** [review]
LGTM 👍

**Comment by ogenev** [comment]
> However, I don't feel too strongly about refactoring `ethportal-peertest` but these seem like endpoints that should be tested. I'm also happy if you want to do it in a following pr.

I think we definitely need to revisit and refactor `ethportal-peertest` to support multiple networks and add beacon and state 
tests.

**Comment by morph-dev** [comment]
I added some tests to `ethportal-peertest` and did a bit of refactoring.

More refactoring would be needed for future tests. And probably with a different approach (because content key/value types are different).

**Review by njgheorghita — APPROVED** [review]
Nice! Thanks for the improvements to `peertest`!

**Pull request merged**